guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc

```diff
@@ -11,12 +11,12 @@ endif::[]
 * The host that you want to edit must be registered with the subscription-manager.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *Content Hosts* and click the name of the {RHEL} 8 host that you want to edit.
-. In the *System Purpose* area, click the *Edit* or *Remove* icon for the system purpose attributes that you want to edit, add, or remove.
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the name of the hosts you want to modify.
+. Click *Edit* next to the *System purpose* card.
+. Select the purpose attributes for your host.
 . Click *Save*.
-. Click the *Subscriptions* tab and select *Subscriptions*.
-. Click *Run Auto-Attach* to attach subscriptions to your host automatically.
-. Refresh the page to verify that the subscriptions list contains the correct subscriptions.
+// Is there a way to auto-attach subscriptions here?
```

Comment:
I don't know the answer to this question so you will either have to tag someone specifically or ask an SME.

Reply:
> I don't know the answer to this question so you will either have to tag someone specifically or ask an SME.

Yeah, I’m adding a few procedure and tagging SME + QA. Thanks for the feedback!

Reply:
No, there isn't an auto-attach button here, because most users will eventually be using SCA and not need it. Users who need to run auto-attach will need to visit the legacy UI. See also my comment above.